charts/opentelemetry-collector/values.yaml

```diff
@@ -8,6 +8,28 @@ fullnameOverride: ""
 # Valid values are "daemonset" and "deployment".
 mode: ""
 
+# Handles basic configuration of components that
+# also require k8s modifications to work correctly.
+# .Values.config can be used to modify/add to a preset
+# component configuration, but CANNOT be used to remove
+# preset configuration. If you require removal of any
+# sections of a preset configuration, you cannot use
+# the preset. Instead, configure the component manually in
+# .Values.config and use the other fields supplied in the
+# values.yaml to configure k8s as necessary.
+presets:
+  # Configures the collector to collect logs.
+  # Adds the filelog receiver to the logs pipeline
+  # and adds the necessary volumes and volume mounts.
+  logsCollection:
+    enabled: false
+    includeCollectorLogs: false
+  # Configures the collector to collect host metrics.
+  # Adds the hostmetrics receiver to the metrics pipeline
+  # and adds the necessary volumes and volume mounts.
```

Comment:
Please mention that it should be used with deamonset mode only

Reply:
Done.  Based on [this conversation](https://github.com/open-telemetry/opentelemetry-helm-charts/pull/294#discussion_r933594388), I went with `Best used with ...`.